features/theme-update.feature

```diff
@@ -113,3 +113,45 @@ Feature: Update WordPress themes
       """
       Error: Can't find the requested theme's version 1.4.2 in the WordPress.org theme repository (HTTP code 404).
       """
+  
+  Scenario: Error when both --minor and --patch are provided
+    Given a WP install
+
+    When I try `wp theme update --patch --minor --all`
+    Then STDERR should be:
+      """
+      Error: --minor and --patch cannot be used together.
+      """
+    And the return code should be 1
+
+  Scenario: Update a theme to its latest minor release
+    Given a WP install
+    And I run `wp theme install --force twentytwelve --version=2.5.4`
```

Comment:
There is no version 2.5.4 or 2.6.1 of Twenty Twelve 🤔 

See https://themes.trac.wordpress.org/browser/twentytwelve/

We could use 3.0 with an expected update to 3.9 though

Reply:
@swissspidy I have changed version as you mentioned but still don't know why test cases are failing can you please   help me with this?

Reply:
See comment below